views/package_suppliers.dt

```diff
@@ -0,0 +1,37 @@
+extends layout
+
+block title
+	- auto title = "Package suppliers";
+
+block body
+	h2 Introduction
+
+	p Dub supports package suppliers of type Dub Registry, Maven and File System. By default Dub will search on Dub Registry <a href="https://code.dlang.org">https://code.dlang.org</a> and various fallbacks.
```

Comment:
If the list of fallbacks is somewhere documented I would also link to that, otherwise specify "hard-coded and user-set fallbacks".

Reply:
fixed